tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/ExtensionsMetadataGenerator.cs

```diff
@@ -47,6 +50,8 @@ public static void Generate(string sourcePath, string outputPath, Action<string>
             File.WriteAllText(outputPath, json);
         }
 
+        public static bool AssemblyShouldBeSkipped(string fileName) => fileName.StartsWith("System", StringComparison.OrdinalIgnoreCase) || ExcludedAssemblies.Contains(fileName);
```

Comment:
`Contains` will perform a case sensitive comparison here. Can you use the overload that takes an equality comparer so this is case insensitive?